common/src/main/java/org/dash/wallet/common/ui/dialogs/AdaptiveDialogExt.kt

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2022 Dash Core Group.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.dash.wallet.common.ui.dialogs
+
+import android.os.Bundle
+import android.view.View
+import android.widget.TextView
+import androidx.annotation.DrawableRes
+import androidx.annotation.LayoutRes
+import androidx.fragment.app.FragmentActivity
+import org.dash.wallet.common.R
+
+class AdaptiveDialogExt(@LayoutRes private val layout: Int) : AdaptiveDialog(layout) {
```

Comment:
`layout` should have a default value to avoid a crash from recreating this dialog